app/src/main/java/com/github/se/project/ui/navigation/NavigationActions.kt

```diff
@@ -45,12 +46,12 @@ data class TopLevelDestination(val route: String, val icon: ImageVector, val tex
 // Top-level destinations with icons
 object TopLevelDestinations {
   val HOME_TUTOR =
-      TopLevelDestination(route = Route.HOME, icon = Icons.Outlined.Home, textId = "Home")
+      TopLevelDestination(route = Screen.HOME, icon = Icons.Outlined.Home, textId = "Home")
   val HOME_STUDENT =
-      TopLevelDestination(route = Route.HOME, icon = Icons.Outlined.Home, textId = "My Courses")
+      TopLevelDestination(route = Screen.HOME, icon = Icons.Outlined.Home, textId = "My Courses")
```

Comment:
Are you sure we should use Screen instead of Route for the top level destinations? It's no big deal as those two have the same string value, but still conceptually haha!